spec/support/vcr_multipart_matcher_helper.rb

```diff
@@ -0,0 +1,21 @@
+#This is needed for mismatching multi-part boundaries -- replacing random boundaries
+#Multipart boundaries are most often randomly generated by the HTTP library or browser
+#so when VCR tries to match a multipart request with a cassette, the boundaries will be different and the match will fail.
+#This was introduced when upgrading to Faraday 0.17.0
+
+#ex -----------boundary=RubyMultipartPost-some_random_boundary_string
```

Comment:
Love the comment, but could you expand on this example a bit - maybe provide a before and after of what it looks like once `normalized_multipart_request` runs?

Reply:
Done ✔️